src/System.Net.Http/src/System/Net/Http/Managed/AuthenticationHelper.Digest.cs

```diff
@@ -183,7 +176,7 @@ internal partial class AuthenticationHelper
 
             string response = ComputeHash(ComputeHash(a1, algorithm) + ":" +
                                         nonce + ":" +
-                                        digestResponse.NonceCount.ToString("x8") + ":" +
+                                        DigestResponse.NonceCount.ToString("x8") + ":" +
```

Comment:
If NonceCount is a const, why use ToString to format it every time?  The string should just be a constant.